tests/test_deployments.py

```diff
@@ -90,8 +92,16 @@ def test_update_deployment(address_or_alias, abi, expected_lines):
         lines = fp.readlines()
     assert len(lines) == 3
 
+    # make logs visible to test
+    logging.getLogger().setLevel(logging.INFO)
+
     update_abi(address_or_alias, abi, LOCALHOST)
 
+    identifier = address_or_alias
+    if type(address_or_alias) == int:
+        identifier = hex_address(address_or_alias)
+    assert f"Updating {identifier} in {LOCALHOST}.{DEPLOYMENTS_FILENAME}"
```

Comment:
couldn't `identifier` be an alias? the new implementation only logs hex addresses

Reply:
Good catch! Opened https://github.com/OpenZeppelin/nile/pull/292 to fix this.